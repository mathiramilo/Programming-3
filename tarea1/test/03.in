leer_grafo
10 dirigido
1: 2 3 4 5 6 .
2: 3 4 5 6 .
3: 4 5 6 .
4: 5 6 .
5: 6 .
6: .
0: .
orden_topologico
leer_grafo
10 dirigido
1: 2 3 4 5 6 .
2: 1 3 4 5 6 .
3: 1 2 4 5 6 .
4: 1 2 3 5 6 .
5: 1 2 3 4 6 .
6: 1 2 3 4 5 .
0: .
orden_topologico
Fin
