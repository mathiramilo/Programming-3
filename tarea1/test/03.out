1>2>Hay orden topologico 
3>4>Hay un ciclo 
5>Fin.
